test/20_royalties_tests.ts

```diff
@@ -540,4 +550,216 @@ describe('HolographRoyalties Contract', async function () {
 
     it('should allow inherited contract to call fn');
   });
+
+  describe('Royalties Distribution Validation', () => {
+    const errorMargin = ethers.utils.parseUnits('2000', 'wei');
+
+    describe('A collection with 1 recipient', async () => {
+      const totalRoyalties = ethers.utils.parseEther('1');
+
+      before(async () => {
+        const data = (await royalties.populateTransaction.configurePayouts([owner.address], [10000])).data || '';
+        const tx = await l1.factory.connect(owner).adminCall(royalties.address, data);
+        await tx.wait();
+      });
+
+      it('should be able to withdraw all native token balance', async () => {
+        await owner.sendTransaction({
+          to: royalties.address,
+          value: totalRoyalties,
+        });
+
+        const accountBalanceBefore = await ethers.provider.getBalance(owner.address);
+        const contractBalanceBefore = await ethers.provider.getBalance(royalties.address);
```

Comment:
We should have an expect check for `contractBalanceBefore` that is tied to the `totalRoyalties` balance

Reply:
Done